proposal/forward-with.md

```diff
@@ -0,0 +1,170 @@
+# Reconfigurable Modules via "@forward ... with": Draft 1
```

Comment:
I'd probably just call this either "Reconfigurable Modules" or "`@forward ... with`" for terseness.